server/nunjucks/nunjucksFunctions.ts

```diff
@@ -176,6 +183,12 @@ export const allowedImageFileTypeLabelList = () =>
     allowedImageFileExtensions.map(ext => ext.label),
     'or'
   )
+export const allowedNoteFileExtensionList = () => allowedNoteFileExtensions.map(ext => ext.extension).join(',')
```

Comment:
We can refactor the existing method to support this without duplicating, but later